doc/man.rb

```diff
@@ -40,7 +40,7 @@
 class ManPageConverter < Asciidoctor::Converter::ManPageConverter
 
   def convert_inline_monospaced node
-    %[#{ESC_BS}fI<BOUNDARY>#{node.text}</BOUNDARY>#{ESC_BS}fP]
```

Comment:
I don't know if this removal is breaking anything, but those tags popped up on the man page and it was impossible to read.

Reply:
I see, thanks for the explanation. If it turns out that another safeguard is needed here it shouldn't be much of a problem to add it back.